core/dbt/include/global_project/macros/materializations/models/table/create_table_as.sql

```diff
@@ -27,7 +27,7 @@
     {{ relation.include(database=(not temporary), schema=(not temporary)) }}
   {% if config.get('contract', False) %}
     {{ get_assert_columns_equivalent(sql) }}
-    {{ get_columns_spec_ddl() }}
+    {{ get_table_columns_and_constraints() }}
```

Comment:
I like this name a lot more. But we'll also need to update that macro in each of the adapters, correct?

Reply:
That's right. It's a new macro in 1.5, though, so it should be contained only to our officially supported adapters.